auditbeat/module/auditd/audit_linux.go

```diff
@@ -980,7 +980,7 @@ func determineSocketType(c *Config, log *logp.Logger) (string, error) {
 			return "", fmt.Errorf("failed to create audit client: %w", err)
 		}
 		// Ignore errors if a socket type has been specified. It will fail during
-		// further setup and its necessary for unit tests to pass
+		// further setup and is necessary for unit tests to pass.
```

Comment:
I'm still not clear what this actually means.

Reply:
I agree. Should we maybe remove that second unclear sentence?

Reply:
Looking through the code path, I think I understand what is being said here; that we are here to determine the type, but we've already been told it, so if we fail it doesn't matter, and there is a test that depends on this returning nil (this is `TestConfigValidateImmutable` with the socket type "unicast"), so we must let this through. I don't think we are losing much by removing the second sentence.

Reply:
Second sentence removed.  97d5ab5